packages/web/src/pages/sign-up-page/pages/CreateEmailPage/CreateEmailPageMobile.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Button,
+  ButtonType,
+  Divider,
+  Flex,
+  IconArrowRight,
+  IconAudiusLogoHorizontalColor,
+  SocialButton,
+  Text
+} from '@audius/harmony'
+
+import { HarmonyTextField } from 'components/form-fields/HarmonyTextField'
+import { Link } from 'components/link'
+import { MobileContentContainer } from 'pages/sign-on/components/desktop/MobileContentContainer'
+import {
+  ArtworkContainer,
+  AudiusValues,
+  SignOnContainerMobile
+} from 'pages/sign-on/components/desktop/PageWithAudiusValues'
+import { SIGN_IN_PAGE } from 'utils/route'
+
+import styles from './CreateEmailPage.module.css'
+import { messages } from './messages'
+
+export const CreateEmailPageMobile = ({
+  isSubmitting
+}: {
+  isSubmitting: boolean
+}) => {
+  return (
+    <SignOnContainerMobile>
+      <ArtworkContainer isDesktop={false}>
+        <MobileContentContainer
+          gap='2xl'
+          alignItems='center'
+          className={styles.mobileCurvedContainer}
+        >
+          <IconAudiusLogoHorizontalColor />
+          <Flex
+            direction='column'
+            gap='s'
+            alignItems='center'
+            w='100%'
+            className={styles.textAlignCenter}
```

Comment:
css prop, obvs whatever you want to do though :)

Reply:
Updated!